Sync/SyncTelemetry.swift

```diff
@@ -89,14 +89,14 @@ public struct ValidationStats: Stats {
 
 public class StatsSession {
     var took: Int64 = 0
-    var when: Int64?
+    var when: Timestamp?
 
-    private var startUptime: Int64?
+    private var startUptime: UInt64?
 
-    public func start(when: Int64 = Int64(Date.now())) {
+    public func start(when: UInt64 = Date.now()) {
         // Convert to milliseconds as stated in the ping format
         self.when = when / 1000
```

Comment:
Wait a minute.

```
    public static func now() -> Timestamp {
        return UInt64(1000 * Date().timeIntervalSince1970)
    }

    public static func nowMicroseconds() -> MicrosecondTimestamp {
        return UInt64(1000000 * Date().timeIntervalSince1970)
    }
```

`when`, if not specified, is already milliseconds. Why are you dividing by 1000?